src/extensions/default/InlineColorEditor/main.js

```diff
@@ -27,7 +27,9 @@ define(function (require, exports, module) {
     var EditorManager       = brackets.getModule("editor/EditorManager"),
         ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
         InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
-        ColorUtils          = brackets.getModule("utils/ColorUtils");
+        ColorUtils          = brackets.getModule("utils/ColorUtils"),
+	 CSSProperties       = require("text!CSSProperties.json"),
+        properties          = JSON.parse(CSSProperties);
```

Comment:
This will will throw if that file has any syntax errors.  I think you should do this in a `try` block on the first instance where you need it.